proxies/go/handler_command.go

```diff
@@ -22,11 +25,31 @@ type CommandBody struct {
 	Params  []Param            `json:"params"`
 }
 
+type LocationResponse struct {
+	EntityType string      `json:"entityType"`
+	Data       interface{} `json:"data"`
```

Comment:
we should be consistent about using `any` vs `interface{}` (pretty sure they're the same thing)

personally I prefer `any` because it's more clear what it means (especially to all us Typescript folks)